policy-controller/src/main.rs

```diff
@@ -72,6 +74,9 @@ struct Args {
     #[clap(long, default_value = "all-unauthenticated")]
     default_policy: DefaultPolicy,
 
+    #[clap(long, default_value = "linkerd-destination")]
+    policy_deployment_name: String,
```

Comment:
is there anything in e.g. the helm chart that ensures this flag is overridden with the actual deployment name? should there be? do we actually expect the deployment will ever be renamed?

Reply:
I think the answers are: no, no, and no. We don't specify this flag in the helm template because the default is correct.  Having this as a flag, however, means that if we do rename the deployment, we'd be able to make that change as a Helm template change only without needing to make a code change here.  Not a very practical concern, but this felt more correct than hardcoding it.

Reply:
cool, just curious about the rationale. that makes sense to me!